src/main/webapp/script.js

```diff
@@ -25,36 +25,36 @@ import tippy, { sticky } from 'tippy.js';
 import 'tippy.js/dist/tippy.css';
 import 'tippy.js/dist/backdrop.css';
 import 'tippy.js/animations/shift-away.css';
+import { colorScheme, opacityScheme } from './constants.js';
 
 export {
-  searchNode, initializeNumMutations, setCurrGraphNum, initializeTippy,
-  generateGraph, getUrl, navigateGraph, currGraphNum, numMutations, updateButtons,
-  highlightDiff, initializeReasonTooltip, getGraphDisplay
+  initializeNumMutations, setMutationIndexList, setCurrMutationNum, initializeTippy,
```

Comment:
is there any way to simplify this long list of functions or work around it like just exporting the whole file? at this point, it is getting difficult to read and use

Reply:
what do you suggest we could do?